### ISSUE TITLE
Compilation using pproj fails on MacOS X on case sensitive volumes.

### ISSUE THREAD

**Issue by anchakarov** [p-org/P#310]
Hi, I am following the TwoPhaseCommit example.

```
% pc -proj:TwoPhaseCommit.pproj
----------------------------------------
==== Loading project file: TwoPhaseCommit.pproj
<Internal Error>:
 Could not find a part of the path '/volumes/workplace/p/src/pexamples/twophasecommit/twophasecommit.pproj'.
 <Please report to the P team (p-devs@amazon.com) or create a issue on GitHub, Thanks!>
------------------------------------------
```

For some reason the path gets renamed to all lower case and as a result compilation fails on a case sensitive volume under Mac OS X 10.15.7.
For information even ls fails on the volume if the proper case is ignored.

**Comment by anchakarov** [comment]
Compilation succeeds if I duplicate all project files and folders with names in all lower case.

**Comment by ankushdesai** [comment]
This is a known (pending issue), thanks for bringing it up again. I will fix it and get back to you.

**Comment by ankushdesai** [comment]
@anchakarov this issue should be fixed in the next release of P. Thanks!

**Comment by ankushdesai** [comment]
This should be fixed in the latest release of P 1.0.8. Thanks!!
